implementation/elasticsearch/src/main/java/net/opentsdb/meta/NamespacedAggregatedDocumentQueryBuilder.java

```diff
@@ -207,9 +206,12 @@ FilterBuilder getMetricFilter(final MetricFilter filter, final boolean nested) {
     }
   }
 
-  FilterBuilder getTagValueFilter(final TagValueFilter filter, final boolean nested) {
+
+
+  FilterBuilder getTagValueFilter(final TagValueFilter filter, final boolean nested) throws Exception {
     if (filter instanceof TagValueLiteralOrFilter) {
       // handles the range filter as well.
+      filter.initialize(null).join();
```

Comment:
Could you toss the Exceptions in the signatures and just catch and throw a new QueryException please?